files/conf/docker-daemon.json

```diff
@@ -3,5 +3,10 @@
         {{- range $i, $e := .RegistryMirrors }}
         {{- if ne $i 0 }},{{ end }}{{ "\n" }}        "https://{{ $e }}"
         {{- end }}
-    ]
+    ],
+    "metrics-addr" : "0.0.0.0:9323",
```

Comment:
Should be fine but please test it. When you create a tenant cluster with this change, then you should still be able to curl this port from the node.

Reply:
I will try following https://gigantic.slack.com/archives/CHAHY6VMF/p1595338496088700